CONTRIBUTING.md

```diff
@@ -0,0 +1,122 @@
+## Contributing to Gopherize
+
+First off, thanks for taking the time to contribute!! ❤️
+
+### Quick Reference
+
+I want to...
+
+_add an exercise! ➡️ [read this](#addex) and then [open a Pull Request](#prs)_
+
+_update an outdated exercise! ➡️ [open a Pull Request](#prs)_
+
+_report a bug! ➡️ [open an Issue](#issues)_
+
+_fix a bug! ➡️ [open a Pull Request](#prs)_
+
+_implement a new feature! ➡️ [open an Issue to discuss it first, then a Pull Request](#issues)_
+
+<a name="#src"></a>
+### Working on the source code
+
+`cmd/root.go` contains a simple `gopherize` CLI that connects to most of the other source files.
+
+<a name="addex"></a>
+### Adding an exercise
+
+The first step is to add the exercise! Name the file `exercise/yourTopic/yourTopicN.go`, make sure to
```

Comment:
The file format of the exercise should be as follows:

If multiple exercises exist for 1 topic -> exercise/yourTopic/yourTopic1/main.go
if only a single exercise -> exercise/yourTopic/yourTopic/main.go
You can refer to intro exercise

Reply:
Updated